charts/merlin/ci/ci-values.yaml

```diff
@@ -1,13 +1,17 @@
 global:
   oauthClientID: "global-client-123"
   mlp:
+    fullnameOverride: "mlp"
     apiPrefix: ""
     serviceName: mlp
     externalPort: "8080"
     vsPrefix: "/api"
     useServiceFqdn: true
     uiPrefix: "/"
     uiServiceName: mlp
+  deployment:
```

Comment:
Rather than this, should we make the merlin chart change in a separate PR so that we can directly consume MLP chart 0.6.2?

Reply:
I will remove the MLP changes from this PR and create another one